railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -326,6 +326,10 @@ def create_system_test_files
         build(:system_test) if depends_on_system_test?
       end
 
+      def create_storage_files
+        build(:storage) unless skip_active_storage?
```

Comment:
thanks i just removed `#delete_storage_files_skipping_active_storege` as it isn't needed